src/Sotsera.Blazor.Toaster/Core/ToastElement.razor

```diff
@@ -4,21 +4,35 @@
 *@
 @inherits ToastElementModel
 @Css
-<div class="@Toast.State.ToastClass" style="@Toast.State.AnimationStyle" @onmouseenter="MouseEnter" @onmouseleave="MouseLeave" @onclick="Clicked">
```

Comment:
I must review the toast class and see why I made the Options class available from both the Options property and the State.Options property.. I changed the project so many times during the asp.net beta releases that I really messed up here. I'll open an issue for this and we'll have to uniform the Options usage in this component.

Reply:
Sounds like a good idea.